newpr.py

```diff
@@ -205,6 +206,15 @@ def get_page_content(self, url):
         except urllib2.URLError:
             return None
 
+    def get_issues(self, labels):
+        suffix = ('+label:' + '+label:'.join(labels)) if labels else ''
+        result = self.api_req("GET", self.open_issues_url % (self.owner, self.repo) + suffix)
```

Comment:
I'm not sure if we want to only search for open issues, since it is possible that an intermittent failure/crash resurfaced in the PR, and we'd want to keep track of that as well.